app/views/lessons/_new.html.erb

```diff
@@ -1,6 +1,3 @@
-<%= form_for([category, @lesson]) do |f| %>
-    <%= hidden_field_tag :current_user, current_user.id %>
-    <div class="btn-group pull-right">
-        <%= f.submit "Let's Start!", class:"btn btn-warning mt-4 btn-lg"%>
-    </div>
-<% end %>
\ No newline at end of file
+
```

Comment:
Why do you still need this partial file? You can just use `<%= render "form"  %>` and delete this file.